test/Misc/expression_too_complex.swift

```diff
@@ -1,4 +1,4 @@
-// RUN: %target-parse-verify-swift -solver-memory-threshold 5000
+// RUN: %target-parse-verify-swift -solver-memory-threshold 4000
```

Comment:
The reasoning is that the 5000 threshold was sufficient memory to process the expression.  Reducing it to 4000 allows the test to pass on ARM, and it still passes on x86_64.